test/files/run/implicit-class-implicit-param-with-default.scala

```diff
@@ -0,0 +1,17 @@
+object Test {
+  implicit class C(self: String)(implicit val foo: String = "default") {
+    def test = foo
+  }
+
+  implicit class WorkaroundOk(self: String)(implicit val foo: String) {
```

Comment:
why is this class in the test? i also don't see how this workaround would work, as far as i understand the compiler only generates an implicit def corresponding to the primary constructor.

Reply:
You're right, this was a work-in-progress-workaround :)

I'll remove from the test.